website/.gitignore

```diff
@@ -1,23 +0,0 @@
-_site
-.doctrees
-.sass-cache
-.jekyll-metadata
-vendor/
-.bundle/
-content/
-
-# Ignore IntelliJ files.
-.idea/
-*.iml
-*.ipr
-*.iws
-
-# Hugo
-www/node_modules
-www/dist
-www/site/resources
-www/site/code_samples
-www/site/_config_branch_repo.toml
-www/yarn-error.log
-!www/site/content
-
```

Comment:
This is a change that happened on master that's being duplicated here, please merge the website-revamp branch into this PR.